Day-I-Social-Card/social-card/public/index.html

```diff
@@ -22,6 +22,10 @@
     <title>React App</title>
   </head>
   <body>
+      <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
```

Comment:
In case there is any confusion, you do not need to include these script tags for this project.